lua/lazyvim/plugins/extras/lang/scala.lua

```diff
@@ -0,0 +1,68 @@
+-- NOTE: This setup does not define any lsp or dap specific key bindings
+-- It is recommended that the LazyVim extras dap-core is imported "lazyvim.plugins.extras.dap.core"
+-- If you like you can setup your own key bindings.
+-- For minimalistic setup have a look at https://github.com/scalameta/nvim-metals/discussions/39
+return {
```

Comment:
Is missing `nvim-treesitter` configuration:
```
  {
    "nvim-treesitter/nvim-treesitter",
    opts = function(_, opts)
      vim.list_extend(opts.ensure_installed, { "scala" })
    end,
  },

```

Reply:
Thanks for the hint. 

Until now I have just used the syntax highlighting from Metals.

I agree that using the treesitter highlighting will provide a better experience for most users.

[01f3bbf](https://github.com/LazyVim/LazyVim/pull/1347/commits/01f3bbf9e64e8c1fe2a25cc1d83c02a60bb2d0a9)